doc/classes/NavigationAgent2D.xml

```diff
@@ -158,8 +158,8 @@
 		<member name="debug_path_custom_color" type="Color" setter="set_debug_path_custom_color" getter="get_debug_path_custom_color" default="Color(1, 1, 1, 1)">
 			If [member debug_use_custom] is [code]true[/code] uses this color for this agent instead of global color.
 		</member>
-		<member name="debug_path_custom_line_width" type="float" setter="set_debug_path_custom_line_width" getter="get_debug_path_custom_line_width" default="-1.0">
-			If [member debug_use_custom] is [code]true[/code] uses this line width for rendering paths for this agent instead of global line width.
+		<member name="debug_path_custom_line_width" type="float" setter="set_debug_path_custom_line_width" getter="get_debug_path_custom_line_width" default="-1.0" deprecated="Deprecated and has no effect.">
+			Deprecated.
```

Comment:
IMO deprecated APIs in principle should still be functional, they are just flagged as not recommended anymore (and under the risk of later removal). An API that's outright removed/unsupported would be "obsoleted", and that's not a concept we use because at that point we might as well just remove it.

While looking for deprecation vs obsolescence definitions though, I found https://github.com/mdn/content/blob/main/files/en-us/mdn/writing_guidelines/experimental_deprecated_obsolete/index.md?plain=1#L38-L50 where the descriptions match my understanding but it seems they also stopped using "obsolete".

Reply:
```suggestion
		<member name="debug_path_custom_line_width" type="float" setter="set_debug_path_custom_line_width" getter="get_debug_path_custom_line_width" default="-1.0" deprecated="Deprecated and has no effect.">
			This property does nothing.
```
And, instead of "Deprecated and has no effect." it needs an actual proper explanation as to why that is, as well as providing alternatives.

Reply:
Then, as the worst case scenario, omit the message, like `deprecated=""`. It's not necessary.

Reply:
done